Resources/Prototypes/_DV/Entities/Mobs/Player/silicon.yml

```diff
@@ -119,3 +119,39 @@
     state: std_mod
   - type: SiliconLawProvider
     laws: StreamerLaws
+
+- type: entity
+  id: MedicalLawCircuitBoard
+  parent: BaseElectronics
+  name: law board (Medical)
+  description: An electronics board containing the Medical lawset.
```

Comment:
make a base law board and change them all to use it instead of copy pasting the sprite 20 times